ballerina-to-openapi/src/main/java/io/ballerina/openapi/service/mapper/OpenAPIResourceMapper.java

```diff
@@ -287,9 +301,10 @@ private String generateRelativePath(FunctionDefinitionNode resource) {
         relativePath.append("/");
         if (!resource.relativeResourcePath().isEmpty()) {
             for (Node node: resource.relativeResourcePath()) {
-                if (node instanceof ResourcePathParameterNode pathNode) {
+                if (node instanceof ResourcePathParameterNode) {
```

Comment:
```suggestion
                if (node instanceof ResourcePathParameterNode pathNode) {
```
we can remove the cast in line 305.